docs/migration/fastapi.rst

```diff
@@ -174,8 +174,290 @@ and to easily access dependencies from higher levels.
 
     * :doc:`/usage/dependency-injection`
 
+Lifespan
+~~~~~~~~
+
+If you're using an async context manager and pass parameters to it, most likely the order of parameters is inversed between FastAPI and Litestar.
+
+.. tab-set::
+
+    .. tab-item:: FastAPI
+        :sync: fastapi
+
+        .. code-block:: python
+
+            @asynccontextmanager
+            async def lifespan(
+                _app: FastAPI,
+                app_settings: AppSettings,
+            ):
+                # Setup code here
+                yield
+                # Teardown code here
+
+    .. tab-item:: Litestar
+        :sync: litestar
+
+        .. code-block:: python
+
+            @asynccontextmanager
+            async def lifespan(
+                app_settings: AppSettings,
+                _app: Litestar,
+            ):
+                # Setup code here
+                yield
+                # Teardown code here
+
+~~~~~~~
+Cookies
+~~~~~~~
+
+While with FastAPI you usually set cookies on the response `Response` object, in Litestar it will be set at the decorator level.
```

Comment:
Should probably add that (and how) you can set cookies on the response as well

Reply:
I added a link to dynamic cookies if that's ok for you